scripts/core/exp-command-buffer.yml

```diff
@@ -44,6 +77,33 @@ etors:
     - name: COMMAND_BUFFER_ENQUEUE_EXP
       desc: Event created by $xCommandBufferEnqueueExp
       value: "0x1000"
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Command-buffer query information type"
+class: $xCommandBuffer
+name: $x_exp_command_buffer_info_t
+typed_etors: True
+etors:
+    - name: REFERENCE_COUNT
+      desc: |
+            [uint32_t] Reference count of the command-buffer object.
+            The reference count returned should be considered immediately stale.
+            It is unsuitable for general use in applications. This feature is provided for identifying memory leaks.
+
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Command-buffer command query information type"
+class: $xCommandBuffer
+name: $x_exp_command_buffer_command_info_t
+typed_etors: True
+etors:
+    - name: REFERENCE_COUNT
+      desc: |
+            [uint32_t] Reference count of the command-buffer object.
+            The reference count returned should be considered immediately stale.
+            It is unsuitable for general use in applications. This feature is provided for identifying memory leaks.
+
```

Comment:
nit: I don't think the newlines between the end of the definition and the separator is common elsewhere in the scripts